packages/eslint-plugin/src/rules/no-import-of-dev-dependencies.ts

```diff
@@ -26,16 +26,19 @@ const rule = createRule({
       const devdeps = packageJson
         ? Object.keys(packageJson.devDependencies).map((dep) => dep.replace(/@types\//, ""))
         : [];
+      const deps = packageJson
+        ? Object.keys(packageJson.dependencies ?? {}).map((dep) => dep.replace(/@types\//, ""))
+        : [];
```

Comment:
Preexisting but isn't this wrong when the package is scoped?

Reply:
Yep

Reply:
😬